modules/gdnative/include/gdnative/rect2.h

```diff
@@ -35,19 +35,24 @@
 extern "C" {
 #endif
 
+#include "core/math/math_defs.h"
 #include <stdint.h>
 
+#define GODOT_RECT2_SIZE sizeof(real_t) * 4
+
 #ifndef GODOT_CORE_API_GODOT_RECT2_TYPE_DEFINED
 #define GODOT_CORE_API_GODOT_RECT2_TYPE_DEFINED
 typedef struct godot_rect2 {
-	uint8_t _dont_touch_that[16];
+	uint8_t _dont_touch_that[GODOT_RECT2_SIZE];
 } godot_rect2;
 #endif
 
+#define GODOT_RECT2I_SIZE sizeof(int32_t) * 4
```

Comment:
`Vector2/3i` members are defined as `int`, which is 32-bit on all supported platforms, but it's "at least
16" according to C++ standard.

Reply:
Actually, [Vector3i uses `int32_t`](https://github.com/godotengine/godot/blob/master/core/math/vector3i.h#L46), and I thought Vector2i did the same, but I was mistaken, as [Vector2i uses `int`](https://github.com/godotengine/godot/blob/master/core/math/vector2.h#L268). This inconsistency should probably be fixed.